rero_ils/modules/items/api/record.py

```diff
@@ -362,15 +381,17 @@ def get_item_by_barcode(cls, barcode, organisation_pid):
         :rtype `rero_ils.modules.items.api.api.Item`
         """
         from . import ItemsSearch
+        if not barcode:
+            return None
```

Comment:
I don't understand why you add the possibility to don't set `barcode` parameter and add a check `if not barcode: return None`. Why this parameter isn't required ? Additionally the name of the function is `get_item_by_barcode`, without barcode parameter, it seems strange to use this method...

Reply:
You are right, I based it on the get_patron_by_barcode but the barcode param should be required in both cases.